clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -302,10 +302,16 @@ public void testSerialization() throws Exception {
         checkErrorResponse(createDeleteGroupsRequest(), unknownServerException, true);
         checkResponse(createDeleteGroupsResponse(), 0, true);
         for (short version : LIST_OFFSETS.allVersions()) {
-            checkRequest(createListOffsetRequest(version), true);
-            checkErrorResponse(createListOffsetRequest(version), unknownServerException, true);
+            checkRequest(createListOffsetRequest(version, 1000000L), true);
+            checkErrorResponse(createListOffsetRequest(version, 1000000L), unknownServerException, true);
             checkResponse(createListOffsetResponse(version), version, true);
         }
+        LIST_OFFSETS.allVersions().stream().filter(version -> version >= (short) 7).forEach(
+            version -> {
+                checkRequest(createListOffsetRequest(version, ListOffsetsRequest.MAX_TIMESTAMP), true);
+                checkErrorResponse(createListOffsetRequest(version, ListOffsetsRequest.MAX_TIMESTAMP), unknownServerException, true);
+            }
+        );
```

Comment:
This does not bring much in this test suite. I think that we can remove it. The only important point is to test all the versions here.

Reply:
coolio, have removed it.